docker-compose.yml

```diff
@@ -566,7 +537,7 @@ services:
           - no_proxy
           - MARIADB_VERSION=${MARIADB_VERSION}
       volumes:
-        - ${DATA_PATH_HOST}/mariadb:/var/lib/mysql
+        - ${DATA_PATH_HOST}/mariadb1:/var/lib/mysql
```

Comment:
@denis-chmel probably could be removed. I guess I've added that because I had some problems with db locally. However, is not a big deal though, as for me